requirements-dev.in

```diff
@@ -1,3 +1 @@
 -r requirements.txt
-
-honcho
```

Comment:
We don't need to add supervisor as a dependency: we use it in prod so it's already in `requirements.in`